pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DateTimeFunctions.java

```diff
@@ -642,4 +1138,30 @@ public static long timestampDiff(String unit, long timestamp1, long timestamp2)
     ISOChronology chronology = ISOChronology.getInstanceUTC();
     return DateTimeUtils.getTimestampField(chronology, unit).getDifferenceAsLong(timestamp2, timestamp1);
   }
+  @ScalarFunction(names = {"timestampDiffMVMV", "dateDiffMVMV"})
+  public static long[] timestampDiffMVMV(String unit, long[] timestamp1, long[] timestamp2) {
```

Comment:
We should revisit these method names, they are kind of confusing now

Reply:
Agreed haha. I removed the `MVMV` variant altogether in https://github.com/apache/pinot/pull/10841/commits/39a2fcdad5489ed5bdfdcd095c9c77ffd90d0380 because I think it would be prone to error (users relying on ordering of MV columns).